ee/cli/templates/nginx-core.mustache

```diff
@@ -36,4 +36,21 @@ gzip_proxied any;
 gzip_comp_level 6;
 gzip_buffers 16 8k;
 gzip_http_version 1.1;
-gzip_types text/plain text/css application/json application/x-javascript text/xml application/xml application/xml+rss text/javascript;
+# Compress all output labeled with one of the following MIME-types.
+gzip_types
+	application/atom+xml
+	application/javascript
+	application/json
+	application/rss+xml
+	application/vnd.ms-fontobject
+	application/x-font-ttf
+	application/x-web-app-manifest+json
+	application/xhtml+xml
+	application/xml
+	font/opentype
+	image/svg+xml
+	image/x-icon
+	text/css
+	text/plain
+	text/x-component;
+	# text/html is always compressed by HttpGzipModule
```

Comment:
@gau1991 Some gzip_types is removed in this pull request
```
application/x-javascript 
application/xml+rss
text/xml
text/javascript
```
Refer: https://github.com/rtCamp/easyengine/pull/423/files#diff-5abc45e8b5d14f596f072bcdf84eb47eL39